docs/targets/elasticsearch.md

```diff
@@ -1,75 +1,58 @@
-# Event target for Elasticsearch
+# Event Target for Elasticsearch
 
-This event target receives [CloudEvents][ce] over HTTP and index their payload intoElasticsearch.
+This event Target receives [CloudEvents][ce] over HTTP and indexs their payload into [Elasticsearch][es]
 
 ## Prerequisites
 
-A Elasticsearch cluster and a set of credentials for indexing documents.
+1. If there is not a pre-existing Elasticsearch cluster available, get started quickly by using either [Elastic Cloud on Kubernetes][eck] or [Elastic Cloud][elasticcloud]
 
-You can get started by using:
+1. Retrieve the proper credentials in either of the following forms:
+    * User and password to the Elasticsearch cluster.
+    * An APIKey instead of User and password.
 
-- [Elastic Cloud on Kubernetes][eck]
-- [Elascic Cloud][elasticcloud]
+Consort the [Elasticsearch documentation][docs] for more information on Elasticsearch.
 
-## Create the Elasticsearch target integration
+## Deploying an Instance of the Target
 
-Create the Elasticsearch target integration in 2 steps:
+Open the Bridge creation screen and add a Target of type `Elastic`.
 
-1. Retrieve credentials for Elasticsearch.
-2. Create the Elasticsearch target.
+![Adding a Elasticsearch Target](../images/elastic-target/create-bridge-1.png)
```

Comment:
Please rename the directory to "elasticsearch-target" (Elastic is the company that builds Elasticsearch).